style.css

```diff
@@ -37,6 +42,41 @@ section {
   margin-bottom: 80px;
 }
 
+a:link {
+  color: #396df2;
+}
+
+a:hover {
+  color: coral;
+  text-decoration-line: line-through;
+}
+
+a:visited {
+  color: #7f8cff;
+  text-decoration-line: line-through;
+}
+
+.submit {
+  padding: 16px;
+  border-radius: 16px;
+  background: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  display: inline-block;
+  text-decoration: none;
+  position: relative;
+  border: 1px solid #6070ff;
+  order: 3;
+}
+
+.submit:hover {
+  background-color: #6070ff;
+  color: white;
+}
```

Comment:
- [ ] Great job adding the hover state to all your buttons. Kindly add the pressed state too. You can use the `:active` pseudoclass.
![image](https://github.com/ethred/Portfolio/assets/94551484/6fcf1d71-5c8a-4410-9da5-9b47989c97c8)

Reply:
The initial reviewer pointed out that the implementation of pseudo classes was missing.
The second reviewer, in line with the previous feedback, requested the implementation of the "pressed" state, correctly identified as the ":active" pseudo class.

- [ ] To meet the project requirements, let's implement the ":active" pseudo class for the buttons using the corresponding color (#2230D2).